src/tests/tasks.test.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import Tasks from '../modules/task.js';
+import editTask from '../modules/editToDo.js';
+
+beforeEach(() => {
+  localStorage.clear();
+  localStorage.setItem.mockClear();
+
+  // Setup Document
+  document.body.innerHTML = `
+  <main>
+    <div id="list-title">
+      <h3>Today's To Do</h3>
+      <a class="refreshBtn"><i class="fa-solid fa-arrows-rotate"></i></a>
+    </div>
+    <form id="input-form">
+      <input
+        placeholder="Add to your list..."
+        id="input-task"
+        class="add-list"
+      />
+      <button type="submit"><span class="arrow">&#8617;</span></button>
+    </form>
+
+    <div id="do-list" class="list-items"></div>
+    <p id="message"></p>
+
+    <div class="clear-all">
+      <button type="button" id="clear" class="clearBtn">
+        Clear all completed.
+      </button>
+    </div>
+  </main>
+  `;
+});
+
+describe('Task Tests', () => {
+  it('Should mark a task as completed', () => {
+    // Setup localStorage
+    localStorage.setItem('array', JSON.stringify([]));
+
+    // Import required functions
+    // eslint-disable-next-line
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: